vmdb/app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -51,7 +51,16 @@ def save_container_groups_inventory(ems, hashes, target = nil)
                 []
               end
 
-    save_inventory_multi(:container_groups, ems, hashes, deletes, [:ems_ref], [:container_definitions, :containers, :labels])
+    hashes.each do |h|
+      next if h[:container_node_name].nil?
+      node = ContainerNode.first(
+        :conditions => {:ems_id => ems.id, :name => h[:container_node_name]})
+      h[:container_node_id] = node.nil? ? nil : node.id
```

Comment:
@Fryguy :

    {:ems_ref=>"f8cf1154-d2e6-11e4-a1a9-525400c903c1",
     :name=>"pod01",
     ...
     :container_node=>
      {:ems_ref=>"54070384-d234-11e4-9611-525400c903c1",
       :name=>"node01",
       :creation_timestamp=>"2015-03-24T14:44:51Z",
       :resource_version=>"54369",
       :id=>5},
    ...

When saving:

    last_refresh_error: "ContainerNode(#40335880) expected, got Hash(#672718..."

I suppose there's nothing on the "saving" side that is translating the hash into the ContainerNode object.